packages/trader/src/App/Components/Elements/Modals/MarketUnavailableModal/__tests__/market-unavailable.spec.jsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { render, screen } from '@testing-library/react';
+import MarketUnavailableModal from '../market-unavailable';
+import { mockStore } from '@deriv/stores';
+import TraderProviders from '../../../../../../trader-providers';
+
+const mock_props = {
+    onCancel: jest.fn(),
+    onConfirm: jest.fn(),
+};
+
+describe('MarketUnavailableModal', () => {
+    it('should render modal component', () => {
+        const mock_root_store = mockStore({
+            ui: {
+                disableApp: jest.fn(),
+                enableApp: jest.fn(),
+                is_loading: false,
```

Comment:
```suggestion
```
@kate-deriv These are default values in `mockStore`, You don't need to override them.

Reply:
@farzin-deriv I'll add `is_loading` and few more to the mock store, thank you